packages/models/types/index.d.ts

```diff
@@ -365,6 +365,12 @@ declare module '@appland/models' {
       constructor(on: boolean, defaultValue: boolean, names: string[]);
     }
 
+    export class DeclutterContextNamesProperty extends DeclutterNamesProperty {
+      public depth: number;
+
+      constructor(on: boolean, defaultValue: boolean, names: string[], depth: number);
```

Comment:
These arguments aren't required because they have default values set in the implementation. Marking them as optionally undefined might be a better fit.

Reply:
OK, I was following along the other example types in this file. I've made the depth optional.